src/vellum/workflows/nodes/core/try_node/node.py

```diff
@@ -101,11 +128,15 @@ class Subworkflow(BaseWorkflow):
                 class Outputs(inner_cls.Outputs):  # type: ignore[name-defined]
                     pass
 
-            class WrappedNode(TryNode[StateType]):
-                on_error_code = _on_error_code
-
-                subworkflow = Subworkflow
-
+            WrappedNode = type(
+                cls.__name__,
+                (TryNode,),
+                {
+                    "__module__": f"{inner_cls.__module__}.{inner_cls.__name__}.{ADORNMENT_MODULE_NAME}",
+                    "on_error_code": _on_error_code,
+                    "subworkflow": Subworkflow,
+                },
+            )
```

Comment:
This is the most important block in the PR to review

First some python context. When Python initiates a class in local scope, they use this special `<locals>` module path. We are borrowing from that pattern to introduce `<adornment>`.

Currently, we were emitting the same `WrappedNode` dynamic class back to vellum, so our translator had no idea how to map it back to a node id. Therefore, we needed a dynamic class so that our workflow translator in Vellum knows how to map it back to the PromptNode. Instead of making the class name dynamic, we are making the module path dynamic and downstream of the original wrapped node in order to distinguish which original node was using reject on error toggle.

I chose the word adornment for a few reasons:
- consistency with the UI
- differentiates from "decorator" - not all python decorators return a new class, let alone a new node

Eager for feedback here both on approach and choice